docker/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM centos:7
+MAINTAINER Apache BookKeeper <dev@bookkeeper.apache.org>
+
+ARG BK_VERSION=4.4.0
+ARG DISTRO_NAME=bookkeeper-server-${BK_VERSION}-bin
+ARG ZK_VERSION=3.5.2-alpha
+
+# Download Apache Bookkeeper and zookeeper, untar and clean up
+RUN set -x \
```

Comment:
I think in #197, @caiok 's script verifies the download package. we should verify that md5, sha1 and asc for the downloaded package.